Yoast/wordpress-seo
# Ensure only a single indexable query is run for the posts overview

## Summary


This PR can be summarized in the following changelog entry:

* Prevents duplicate queries for each and every post on the posts overview.

## Relevant technical choices:

* Only a single query is done for all posts.

## Test instructions

This PR can be tested by following these steps:

* Make sure you're running something like query monitor to easily view queries.
* Go to the posts overview.
* Enable the SEO title and meta description columns.
* Refresh the page.
* You should see only a single query being done on the indexables table.
* Go to different pages and try with different columns, in all cases you should see only a single query on the indexables table.

## UI changes

## Documentation

## Quality assurance


Fixes https://yoast.atlassian.net/browse/P2-117
Fixes https://yoast.atlassian.net/browse/P2-120
